pipeline/pipeline.go

```diff
@@ -175,6 +175,9 @@ func (r *Runtime) execAll(steps []*backend.Step) <-chan error {
 				return err
```

Comment:
you dont need to change the default( or any other) tracer as they will be exec here at L173 ... so we always apply to it afterwards in any case :)